test/androidTest/java/org/thoughtcrime/securesms/espresso/ViewActions.java

```diff
@@ -80,6 +58,40 @@ public void perform(UiController uiController, View view) {
         };
     }
 
+    public static ViewAction clickPercent(final float pctX, final float pctY) {
```

Comment:
I think final is unnecessary in this case in the params

Reply:
idk it's the IDE who refactored it, i'll remove